exporter/otlp-grpc/lib/opentelemetry/exporter/otlp/grpc/exporter.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry/common'
+require 'opentelemetry/exporter/otlp/common'
+require 'opentelemetry/sdk'
+require 'grpc'
+
+require 'google/rpc/status_pb'
+require 'opentelemetry/proto/collector/trace/v1/trace_service_services_pb'
+
+module OpenTelemetry
+  module Exporter
+    module OTLP
+      module GRPC
+        # An OpenTelemetry trace exporter that sends spans over GRPC.
+        class Exporter
```

Comment:
Looks like we didn't do:
> Refine the exporter namespace `OpenTelemetry::Exporter::OTLP::HTTP::Exporter` -> `OpenTelemetry::Exporter::OTLP::HTTP::TraceExporter` or `OpenTelemetry::Exporter::OTLP::HTTP::Trace`

from https://github.com/open-telemetry/opentelemetry-ruby/pull/1179#issuecomment-1118988405.

Reply:
Yeah, sorry I will be following up with that after this gets merged.  It's on a _remarkable_ todo list.  None of these new exporters have been published yet so I'm taking that opportunity to carve out some smaller (bikesheddable) changes into their own PRs.